projects/flash-cards.md

```diff
@@ -258,8 +258,116 @@ At the beginning of the game, give options for choosing to study different subje
+
+# Notes on Project Evaluations
+
+The rubric below serves as a guide for students:  
+- It should be used as a reference throughout the project to keep on track and guide learning.
+- It should also be used by students to self-assess their work.
+
+<section class="answer">
+
+### Projects as Learning Tools
+When projects are graded, we want you to view the evaluation + feedback as a means to inform your learning, rather than as static "grades". Feedback from instructors will focus on areas where you have an opportunity to deepen your understanding.
+
+The evaluation will provide feedback by answering the only important question:
+
+**Does the project demonstrate student understanding of the learning goals & concepts?**
+
+Projects will answer that question, being marked as **yes**, **not yet**, and **wow**. Similarly, each section of the rubric (see below) will have yes/not yet/wow markings, helping you understand your progress and growth in specific areas.
+
+The overall project outcome (yes, not yet, wow) is determined by "averaging" each section's outcome. You can think of a "yes" being worth a  1, a "not yet" being worth a 0, and a "wow" being worth a 2.
+
+For this project, an average of 0.5 is considered a yes - a passing project that demonstrates good student understanding! An average of 1+ is considered a wow. Anything below a 0.5 is considered a not yet - a project that indicates that the concepts have not been fully understood (see note in the section below).
+
+** 🚨 A important note about the possible outcomes**
+
+**Yes** indicates that the student/team is ON TRACK in this area! YES you showed us you understand the concept!
+- The student/team demonstrates a good understanding of the concept
+
+**Not Yet** indicates that the student/team is BEHIND in this area - you showed us that you don't yet understand the concept
+- The student/team demonstrates misconceptions or confusion around the topic
+- Student/team should prioritize this concept in their studying and practice
+- ✨ An overall "not yet" on one project does **not** mean the module is doomed! We want to see GROWTH. A student/team can struggle with a project and still (taking into account the rest of their work and assessments) demonstrate readiness for the next module! ✨
+
+**Wow** indicates that the student/team did extra work to teach themselves a new concept, or was able to demonstrate a deep & nuanced understanding of a concept  
+- This is not an outcome to prioritize achieving
+- "Wow" should not come at the expense of another concept
+- "Wow" often does not look like extra features, but instead looks like thoughtful refactoring and polish 💅
+</section>
+
+
 ## Evaluation Rubric
 
+This project has 3 evaluated concepts:
+- Functionality
+- JavaScript & Style
+- Test Driven Development
+
+### Minimum Professionalism Expectations
+Note: Professionalism is not a separately graded competency of this project because by now, it is a standard expectation for all your work.  
```

Comment:
👍🏻

Reply:
I wonder if we can consider making Professionalism un-scored but influencing? Like poor professionalism will round down an overall project score??

Reply:
Typically what I've seen on this one is issues with the README which I usually give them as an action item to correct right away so they see hopefully start to see it as a non-negotiable.